src/components/ChatEntry.js

```diff
@@ -3,20 +3,35 @@ import './ChatEntry.css';
 import PropTypes from 'prop-types';
 
 const ChatEntry = (props) => {
+  const currentYear = new Date().getFullYear();
+  const year = currentYear - new Date(props.timeStamp).getFullYear();
```

Comment:
Ooh, nice. This works since it's always years off, but ideally you'd want to grab a library that does this sort of conversion to relative time. In fact that's exactly what the `TimeStamp` component does using the luxon library. So your work here is redundant with that.

That's not too major an issue, though.